src/components/sbb-navigation/sbb-navigation.tsx

```diff
@@ -345,10 +345,10 @@ export class SbbNavigation implements ComponentInterface, AccessibilityPropertie
 
     // Close navigation on backdrop click
     this._element.addEventListener('pointerdown', this._pointerDownListener, {
-      signal: this._navigationController.signal,
+      signal: this._navigationController?.signal,
```

Comment:
Very weird, because at this point, _navigationController should be defined.

Reply:
Since there's no trigger, the `configure` method exits at line 198.. maybe should we improve tests adding the trigger?